schema/filter_artworks.js

```diff
@@ -23,10 +23,16 @@ export const FilterArtworksType = new GraphQLObjectType({
       type: GraphQLInt,
       resolve: ({ aggregations }) => aggregations.total.value,
     },
+    followed_artists_total: {
+      type: GraphQLInt,
+      resolve: ({ aggregations }) => aggregations.followed_artists.value,
+    },
```

Comment:
Yeah I get that, but I guess that makes sense when there’s only 1 count field. I’m more wondering in general _when_ the best time would be to add a `counts` object. Should we do that even if there’s only 1 count field so that it’s clear where subsequent count fields need to go and we don’t need to move API around?